src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -4133,6 +4133,8 @@ def _parse_major_version_linux(self, major_version, parsed_results):
                     (linux_container_settings.additional_properties.get("java17Runtime"), "17", linux_container_settings.is_auto_update),    # pylint: disable=line-too-long
                     (linux_container_settings.java11_runtime, "11", linux_container_settings.is_auto_update),
                     (linux_container_settings.java8_runtime, "8", linux_container_settings.is_auto_update)]
+                # Remove the JBoss'_byol' entries from the output
+                runtimes = [(r, v, au) for (r, v, au) in runtimes if r is not None and not r.endswith("_byol")]    # pylint: disable=line-too-long
```

Comment:
May I ask if removing JBoss '_byol' entries will cause a breaking change for the customer?